addons/binding/org.openhab.binding.modbus/README.md

```diff
@@ -1030,6 +1030,158 @@ There is no trigger anymore but one can use transformations to accomplish the sa
+    tcp.slave2.length=100
+    tcp.slave2.updateunchangeditems=false
+
+    tcp.slave3.connection=192.168.2.9:502
+    tcp.slave3.type=holding
+    tcp.slave3.id=3
+    tcp.slave3.start=12438
+    tcp.slave3.length=100
+    tcp.slave3.updateunchangeditems=false
+
+    tcp.slave4.connection=192.168.2.9:502
+    tcp.slave4.type=holding
+    tcp.slave4.id=4
+    tcp.slave4.start=12538
+    tcp.slave4.length=100
+    tcp.slave4.updateunchangeditems=false
+```
+
+As you can see, all the slaves poll the same modbus device (actually a Wago 750-841 controller). We now have to create ``Things`` for this pollers.
+
+The 2.x modbus binding uses a three-level definition. Level one defines a ``Bridge``for every modbus slave. The 1.x configuration only has one slave, so there will be one top level bridge.
+
+```
+Bridge modbus:tcp:wago [ host="192.168.2.9", port=502, id=1 ] {
+
+}
+```
+Host and Port are taken from the 1.x modbus config.
+
+Within the top level ``Bridge`` there are one or more second level bridges that replace the former slave configurations. The poll frequency can now be set per slave, so you may want to define different poll cycles up to your needs. The slave ``Bridge`` configs go inside the top level config. For the four pollers defined in this example the 2.x configuration looks like this:
+
+```
+Bridge modbus:tcp:wago [ host="192.168.2.9", port=502, id=1 ] {
+
+    Bridge poller wago_slave1 [ start=12288, length=128, refresh=500, type="coil" ] {
+    }
+
+    Bridge poller wago_slave2 [ start=12338, length=100, refresh=4000, type="holding" ] {
+    }
+
+    Bridge poller wago_slave3 [ start=12438, length=100, refresh=5000, type="holding" ] {
+    }
+
+    Bridge poller wago_slave4 [ start=12538, length=100, refresh=10000, type="holding" ] {
+    }
+}
+```
+
+Address, length and type can be directly taken over from the 1.x config.
+
+The third (and most complex) part is the definition of data ``Thing`` objects for every ``Item`` bound to modbus. This definitions go into the corresponding 2nd level ``Bridge`` definitions. Here it is especially important that the modbus binding now uses absolute addresses all over the place, while the addresses in the item definition for the 1.x binding were relative to the start address of the slave definition before. For less work in the following final step, the update of the ``Item`` configuration, the naming of the data things in this example uses the offset of the modbus value within the poller as suffix, starting with 0(!) 
+
+Here a few examples of the Item configuration from the 1.x binding:
+
+The first Item polled with the first poller used this configuration (with offset 0):
+
+```
+Switch FooSwitch  "Foo Switch"  {modbus="slave1:0"} 
+```
+
+Now we have to define a ``Thing``that can later be bound to that Item.
+
+The ``slave1`` poller uses ``12288`` as start address. So we define the first data Thing within the poller ``wago_slave1`` with this address and choose a name that ends with ``0``:
```

Comment:
Please split lines
single back tick
`poller` instead of plain poller to match documentation elsewhere?
`data` instead of plain data to match documentation elsewhere?